ragna/assistants/_bedrock.py

```diff
@@ -0,0 +1,79 @@
+from typing import cast
+import boto3
```

Comment:
Ignore this if you go for a third party package that can hard-depend on `boto3`. Otherwise, this should be a requirement on the class below and imported locally to avoid introducing a hard runtime dependency.